targets/XPlatCppSdk/source/test/TestApp/PlayFabApiTest.h

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#pragma once
+
+#include <functional>
+#include <string>
+#include "TestCase.h"
+
+namespace PlayFab
+{
+    struct PlayFabError;
+    class PlayFabEventAPI;
```

Comment:
>PlayFabEventAPI [](start = 10, length = 15)

This possibly is not needed in this file

Reply:
Confirmed, doesn't seem necessary.